app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for human in people:
+        person_object = Person(human["name"], human["age"])
+
+        if "wife" in human:
+            if human["wife"] is not None:
+                person_object.wife = human["wife"]
```

Comment:
what is the purpose to do here `person_object.wife = human["wife"]` if you can iterate by `people` instead of `Person.people` in the second loop and get all the needed information?